docs/docs/conf.py/conf.py.txt

```diff
@@ -0,0 +1,10 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# Full list of options can be found in the Sphinx documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+#
+# -- Project information -----------------------------------------------------
+#
+
+project = "sphinx-autobuild"
```

Comment:
These changes are unrelated. Please drop them.

Reply:
I'll make sure to delete the following lines. Thank you.